src/models/UserMapper.php

```diff
@@ -38,6 +38,7 @@ public function getVerboseFields()
             "username"         => "username",
             "full_name"        => "full_name",
             "twitter_username" => "twitter_username",
+            "trusted"          => "trusted"
```

Comment:
There was a reason I exposed it - I think so that we (or indeed a user) can check if they are trusted or not. Otherwise the only way to know would be to dig in the DB, which obviously isn't ideal.